Flutter/circular-charts/right-to-left.md

```diff
@@ -41,56 +41,127 @@ To change the chart rendering direction from right to left, you can change the [
+      return MaterialApp(
+        localizationsDelegates: [
+          GlobalMaterialLocalizations.delegate,
+          GlobalWidgetsLocalizations.delegate,
+        ],
+        supportedLocales: <Locale>[
+          Locale('en'),
+          Locale('ar'),
+          // ... other locales the app supports
+        ],
+        locale: Locale('ar'),
+        home: Scaffold(
+          body: SfCircularChart(
+            //...
+          ),
+        )
+      );
     }
 
-
 {% endhighlight %}
 
 ## RTL supported chart elements
 
-Right to left rendering is effective only for the legend in the chart. Legend items will be rendered from right to left direction.
+### Legend
+
+Right to left rendering is effective for the legend in the chart. Legend items will be rendered from right to left direction.
 
-![legend RTL](images/rtl-support/circular_legend_rtl.png)
+{% highlight dart %}
 
-In addition, if you want to change the tooltip’s content, to look like it is rendering from right to left, then you can set the [`format`](https://pub.dev/documentation/syncfusion_flutter_charts/latest/charts/TooltipBehavior/format.html) property in [`TooltipBehavior`](https://pub.dev/documentation/syncfusion_flutter_charts/latest/charts/TooltipBehavior-class.html) as `point.y : point.x`. By default, the tooltip format will be `point.x : point.y`.
+    @override
+    Widget build(BuildContext context) {
+      final List<ChartData> chartData = <ChartData>[
+        ChartData('Jan', 24),
+        ChartData('Feb', 20),
+        ChartData('Mar', 35),
+        ChartData('Apr', 27),
+        ChartData('May', 30),
+      ];
+      return Scaffold(
+        body: Directionality(
+          textDirection: TextDirection.rtl,
+          child: SfCircularChart(
+            legend: Legend(
+              isVisible: true
+            ),
+            series: <ChartSeries<ChartData, String>>[
+              PieSeries<ChartData, String>(
+                dataSource: chartData,
+                xValueMapper: (ChartData data, _) => data.x,
+                yValueMapper: (ChartData data, _) => data.y,
+              )
+            ]
+          )
+        )
+      );
+    }  
+
+    class ChartData {
+      ChartData(this.x, this.y);
+        final String x;
+        final int y;
+    }
+
+{% endhighlight %}
+
+![Legend RTL](images/rtl-support/circular_legend_rtl.jpg)
+
+### Tooltip
+
+Right-to-left rendering is applicable for [`tooltip`](https://pub.dev/documentation/syncfusion_flutter_charts/latest/charts/TooltipBehavior-class.html) elements. Here, the tooltip content renders at first and followed by that the marker on the right. By default, the tooltip content will be `point.x : point.y`, in RTL rendering, the tooltip content will be `point.y : point.x`. If you wish the format to be applied as it is despite RTL rendering in this case, you can make use of [`onTooltipRender`](https://pub.dev/documentation/syncfusion_flutter_charts/latest/charts/SfCircularChart/onTooltipRender.html) callback.
```

Comment:
Right-to-left rendering is applicable for [`tooltip`](https://pub.dev/documentation/syncfusion_flutter_charts/latest/charts/TooltipBehavior-class.html) elements. Here, the tooltip content renders at first and followed by that the marker on the right. By default, the tooltip content will be `point.x : point.y`, in RTL rendering the tooltip content will be `point.y : point.x`. If you wish the format to be applied as it is despite RTL rendering in this case, you can make use of [`onTooltipRender`](https://pub.dev/documentation/syncfusion_flutter_charts/latest/charts/SfCircularChart/onTooltipRender.html) callback.

Reply:
Removed the comma in all RTL files